naga/src/back/spv/mod.rs

```diff
@@ -466,38 +466,75 @@ enum CachedConstant {
     ZeroValue(Word),
 }
 
+/// How a Naga global variable is represented in SPIR-V.
```

Comment:
style: I strongly suspect this would serve better as a non-punctuated header; suggested edit:


```suggestion
/// # How a Naga global variable is represented in SPIR-V
```

Reply:
The first paragraph of a doc string is used as the summary shown during search. It seems like having a header show up there would be weird?

Reply:
I changed the first paragraph to be a bit less awkward.

Here's how search results look when the summary is an ordinary paragraph:
![global-variable-pre](https://github.com/user-attachments/assets/8cc4977f-275a-400d-9f4b-7f2b553df7d3)

Here's how the page itself looks when the summary is an ordinary paragraph:
![global-var-page-pre](https://github.com/user-attachments/assets/452cf013-045d-41a2-b9c5-d272511ababf)

Here's how search results look when the summary is a header:
![global-variable-post](https://github.com/user-attachments/assets/c2824431-6db9-476f-a537-99131c721ee1)

Basically, there's no difference, except for the missing period.

And here's how the page itself looks when the summary is a header:
![global-var-page-post](https://github.com/user-attachments/assets/86711249-be73-443a-91a2-1b4e19a2903c)

Overall, I think having a summary paragraph be just a paragraph is better.

Reply:
Ahhh, the flavor of the edit to the summary paragraph makes it definitely feel like a summary of the type now, and that feels good. Previously, I thought it had maybe been a contextual phrase for the following content, rather than a summary of what the type is. 🤦🏻‍♂️

Reply:
Yeah, that's fair. I think the "How a..." phrasing is a bit off, for the summary paragraph role.

Actually, it probably shouldn't even be a "paragraph" proper. "Noun clause" is closer. "The SPIR-V representation of a Naga global variable" is a noun clause. "How a ..." was sort of a noun-ification of an action, and I feel it was pushing the boundaries.